storage_v2/depstore/factory.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package depstore
+
+import "github.com/jaegertracing/jaeger/storage_v2"
+
+type Factory interface {
+	storage_v2.FactoryBase
```

Comment:
we should remove this from here and from TraceStore

Reply:
actually, at least from here.

Reply:
removing it altogether doesn't seem to break anything